Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -26,7 +28,7 @@ namespace Camera
         if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serializeContext->Class<CameraComponentConfig, AZ::ComponentConfig>()
-                ->Version(4)
+                ->Version(5)
```

Comment:
It may be good to increase the minor version in `gems/Camera/gems.json` from 0.0.0 to 0.1.0 to account for this new capability